joomla/joomla-cms
# [4.0] Drop GMail Plugin from 4.0 new installs

Pull Request for Issue #18310 .

### Testinstructions

- Install: https://github.com/zero-24/joomla-cms/archive/drop_gmail.zip
- notice there is no gmail plugin anymore.

### Expected result

No GMail Plugin in 4.0 new installs. If needed it can be installed from a joomla-extensions repo.

### Actual result

There is a gmail plugin in the core that is disabled by default + require to enable `allow less secure apps` setting in the google account.

### RFC

Please leave your opinions on this topic below ;)
